diff --git a/cli/src/opts.rs b/cli/src/opts.rs
index bfd801b..73de5ad 100644
--- a/cli/src/opts.rs
+++ b/cli/src/opts.rs
@@ -4,7 +4,7 @@ use clap::{Parser, Subcommand};
 use clap_complete::{
     engine::ValueCompleter, ArgValueCompleter, CompletionCandidate, PathCompleter,
 };
-use std::path::{Path, PathBuf};
+use std::path::PathBuf;
 
 /// Mount backend type
 #[derive(Debug, Clone, Copy, clap::ValueEnum)]
@@ -416,8 +416,7 @@ fn id_completer(current: &std::ffi::OsStr) -> Vec<CompletionCandidate> {
         .filter_map(|e| e.ok())
         .filter_map(|e| {
             let file_name = e.file_name();
-            let path = Path::new(&file_name);
-            let name = path.file_prefix()?.to_str()?;
+            let name = file_name.to_str()?.strip_suffix(".db")?;
             if name.starts_with(current) {
                 Some(CompletionCandidate::new(name).help(Some("Agent ID".into())))
             } else {
